### DISCUSSION TITLE
The dpmd train result becomes bad after open virial tag in loss

### DISCUSSION THREAD

**Discussion by ysu-007** [deepmodeling/dpgen#632] [Q&A]
See the figure for the result. So how to choose the virial parameters to obtain a better result?
force comparsion from dp test
<img width="826" alt="force" src="https://user-images.githubusercontent.com/54423463/150049541-4537e3b4-a62e-4f17-973b-eab7e8785c34.png">
energy comparsion from dp test
<img width="803" alt="energy" src="https://user-images.githubusercontent.com/54423463/150049531-6dcab8b8-7177-474b-9c52-fabe64b3e7e7.png">
Thanks a lot

**Comment by McAlexDudu** [comment]
if your data don't have viral message, just set 0;you can check it by dpdata.
